tests/vocab.ttl

```diff
@@ -43,3 +43,28 @@
   rdfs:comment """
     Canonicalization performed using the RDFC-1.0 algorithm.
```

Comment:
```suggestion
    Canonicalization performed using the RDFC-1.0 algorithm.
    Results are compared as text
    where the result of running the test is serialized to canonical N-Quads, sorted in code point order.
```
I think the following phrase located from line 36 to 37 would be better relocated here because it no longer applies to the test as a whole, but is now only applicable to the EvalTest.

> Results are compared as text where the result of running the test is serialized to canonical N-Quads, lexicographically-sorted.

(However, it appears that I cannot make a suggestion for lines 36-37...)